schemas/process.yml

```diff
@@ -61,9 +61,21 @@
         short_override: An array of previous executions for the process, including the initial fork. Only executable and args are set.
         normalize:
           - array
+      - at: process
+        as: responsible
+        short_override: Information about the responsible process. The responsible process in macOS tracks the true originating process of an application, crucial for understanding permissions and process hierarchy beyond the standard parent process.
```

Comment:
This is failing a test: `ValueError: Short descriptions must be single line, and under 120 characters (current length: 226).`

Could you move this into `description` instead? I think what you had here originally can stay as `short_override`